package.json

```diff
@@ -22,7 +22,7 @@
     "classnames": "^2.2.6",
     "compressorjs": "^1.0.5",
     "detect-browser": "^3.0.1",
-    "masq-common": "git+https://github.com/QwantResearch/masq-common",
+    "masq-common": "git+https://github.com/QwantResearch/masq-common#MASQ-191-hashKeys",
```

Comment:
ok, please merge masq-common before, then use the master branch here before merging this PR

Reply:
ok